src/components/content/order/common/ShowIsv.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { Tag, Tooltip } from 'antd';
+import React from 'react';
+import serviceOrderStyles from '../../../../styles/service-order.module.css';
+
+export const ShowIsv = ({ namespace }: { namespace: string }): React.JSX.Element => {
```

Comment:
rename the component to 'IsvNameDisplay'

Reply:
fixed.